src/components/StatusBar/styles/StatusBar.css.js

```diff
@@ -0,0 +1,100 @@
+import styled from 'styled-components'
+import forEach from '../../../styles/utilities/forEach'
+import { getColor, rgba } from '../../../styles/utilities/color'
+import Centralize from '../../Centralize'
+
+const statusStyles = {
+  error: 'red',
+  info: 'blue',
+  success: 'green',
+  warning: 'yellow',
+}
+
+const statusBarStyleBold = color =>
```

Comment:
What are your thoughts on refactoring these to have explicit returns (as we are doing here: https://github.com/helpscout/hsds-react/pull/750/files#diff-8e4af190189ebd03f97d900f0df4fe5fR38)? The backticks here combined with the implicit return initially confused me as I thought they were styled components.

Reply:
I did update the code to do an explicit return here 0c7afaa5